Source/WebInspectorUI/UserInterface/Views/ConsoleSnippetTreeElement.js

```diff
@@ -86,6 +86,10 @@ WI.ConsoleSnippetTreeElement = class ConsoleSnippetTreeElement extends WI.Script
 
         contextMenu.appendSeparator();
 
+        contextMenu.appendItem(WI.UIString("Delete"), () => {
+            this.ondelete();
```

Comment:
`ondelete` is supposed to only be called when the delete key is pressed, so id prefer this to be inlined as this instead
```js
WI.consoleManager.removeSnippet(this.representedObject);
```